pkg/products/rhssocommon/reconciler.go

```diff
@@ -280,9 +269,56 @@ func (r *Reconciler) SetupOpenshiftIDP(ctx context.Context, serverClient k8sclie
 			},
 		})
 	}
+
+	return nil
+}
+
+// Why do we need to do this?
+func (r *Reconciler) SyncOpenshiftIDPClientSecret(ctx context.Context, serverClient k8sclient.Client, authenticated keycloakCommon.KeycloakInterface, sso config.RHSSOInterface, keycloakRealmName string) error {
+
+	clientSecret, err := r.getClientSecret(ctx, serverClient, sso)
+	if err != nil {
```

Comment:
Should we log the error here?

Reply:
No need here as errors logged in getClientSecret()